src/middlewared/middlewared/plugins/system.py

```diff
@@ -585,22 +585,18 @@ async def _convert_datetime(self, response, timezone_setting=timezone.utc):
     # Sync the clock
     @private
     async def sync_clock(self):
```

Comment:
This should not be async, it performs blocking I/O in `NTPClient`

Reply:
Done